Makefile

```diff
@@ -29,3 +29,12 @@ docker/build:     ## Build all development images.
 .PHONY: docker/test
 docker/test:      ## Run unit tests.
 	./compose run api manage test galaxy_ng.tests.unit
+
+dev/bumpversion-patch:
+	bump2version --verbose patch
+
+dev/bumpversion-minor:
+	bump2version --verbose minor
+
+dev/bumpversion-major:
+	bump2version --verbose major
```

Comment:
Could we have a single make command that is the canonical way to use bump2version? I feel most of the time we will need to specify the entire version string rather than bumping one version section. Also if `new_version` is in the config, I think we need to specify it. This worked for me:
```
bump2version --verbose --new-version=4.3.1.dev part=[]
```


We will likely be using .a# and .b# in the pre-release sections, I tried:
```
bump2version --verbose --new-version=4.3.0.a1 part=[]
```
and `setup.cfg` was changed as expected but `setup.py` and `__init__.py` had invalid version strings: `version = "version = "4.3.0.a""` `__version__ = "__version__ = "4.3.0.a""`

Reply:
`new_version` is generated, so generally no need to specify it. Unless you want/need to bump to an arbitrary version (say, from 4.2.0 -> 5.1.1 etc).

To use '.a#' or .b#' for pre releases, we would need to define that pre-release 'part' ('patch', 'minor', 'major', are default parts and 'release' is defined in the current setup.cfg).

Then you would use 'bumpversion pre-release'. The details of the format are specified with the parse/serialize config for that part.

https://github.com/c4urself/bump2version#optional_value-  has more details. The current config knows about an
optional 'release' for cases like 4.3.0.dev. This is based on what pulp does. 

Haven't tested, but something roughly like:

**UNTESTED** 

``` ini
[bumpversion]
current_version = "4.3.0.dev"
commit = True
tag = True
parse = (?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)(\.(?P<release>[a-z]+))(?P<build>\d+)?
serialize =
	{major}.{minor}.{patch}.{release}{build}
	{major}.{minor}.{patch}

[bumpversion:part:release]
optional_value = stable
first_value = a
values =
	a
        b
	stable

[bumpversion:part:build]
optional_value = 0
first_value = 1

[bumpversion:file:setup.py]
search = version = "{current_version}"
replace = version = "{new_version}"
```

Reply:
And 'bumpversion build' to go from 5.3.7.a1 -> 5.3.7.a2
and 'bumpversion release' to from 5.3.7.a1 -> 5.3.7.b1
and another 'bumpversion release' to from from 5.3.7.b1 -> 5.3.7

I think... untested caveat.

Reply:
Oh, and the main reason for using the Makefile targets is just to have something known. And for me, so I can tab complete 'make dev/<tab>' and figure the command instead of looking it up.

Reply:
I updated this with a config that should handle the cases you mentioned.

Reply:
Also updated so that galaxy_ng/__init__.py and galaxy_ng/app/__init__.py are updated.

Reply:
That sounds good. Can we add something like this in the Makefile?, those examples will be helpful when using in the future

```
# 'bumpversion build' to go from 5.3.7.a1 -> 5.3.7.a2
dev/bumpversion-build:
	bump2version --verbose build

# 'bumpversion release' to from 5.3.7.a1 -> 5.3.7.b1
# another 'bumpversion release' to from from 5.3.7.b1 -> 5.3.7
dev/bumpversion-release:
	bump2version --verbose release
```